cmd/protoc-gen-connect-go/main.go

```diff
@@ -144,12 +168,37 @@ func generatePreamble(g *protogen.GeneratedFile, file *protogen.File) {
 }
 
 func generateServiceNameConstants(g *protogen.GeneratedFile, services []*protogen.Service) {
+	var numMethods int
 	g.P("const (")
 	for _, service := range services {
 		constName := fmt.Sprintf("%sName", service.Desc.Name())
 		wrapComments(g, constName, " is the fully-qualified name of the ",
 			service.Desc.Name(), " service.")
 		g.P(constName, ` = "`, service.Desc.FullName(), `"`)
+		numMethods += len(service.Methods)
+	}
+	g.P(")")
+	g.P()
+
+	if numMethods == 0 {
+		return
+	}
+	wrapComments(g, "These constants are the fully-qualified names of the RPCs defined in this package. ",
+		"They're exposed at runtime as Spec.Procedure and as the final two segments of the HTTP route.")
+	g.P("//")
+	wrapComments(g, "Note that these are different from the fully-qualified method names used by ",
+		"google.golang.org/protobuf/reflect/protoreflect. To convert from these constants to ",
+		"reflection-formatted method names, remove the leading slash and convert the ",
+		"remaining slash to a period.")
```

Comment:
Instead of documenting this in the generated code, is it worth adding a utility function to convert a procedure to a protoreflect.FullName? That seems like a useful method and we could simplify the generated code.

Reply:
I'm hoping to minimize the number of places that the connect-go runtime couples closely to the protobuf runtime. I suspect that the _most common_ reason people would make this translation is to get a method descriptor, so I'm planning to just expose that directly in a subsequent PR.